JS/fluentRevealNavbar.uc.js

```diff
@@ -38,15 +54,20 @@
 
     // get all the toolbar buttons in the navbar, in iterable form
     get toolbarButtons() {
-      let buttons = Array.from(
-        gNavToolbox.querySelectorAll(".toolbarbutton-1")
-      );
-      if (this._options.includeBookmarks) {
-        buttons = buttons.concat(
-          Array.from(this.placesToolbarItems.querySelectorAll(".bookmark-item"))
+      if (!this.buttons || this._options.cacheButtons == false) {
+        this.buttons = Array.from(
+          gNavToolbox.querySelectorAll(".toolbarbutton-1")
         );
+        if (this._options.includeUrlBar) {
+          this.buttons.push(gNavToolbox.querySelector("#urlbar-background"));
+        }
+        if (this._options.includeBookmarks) {
+          this.buttons = this.buttons.concat(
+            Array.from(this.placesToolbarItems.querySelectorAll(".bookmark-item"))
+          );
+        }
```

Comment:
```suggestion
        if (this._options.includeBookmarks) {
          this.buttons = this.buttons.concat(
            Array.from(
              this.personalToolbar.querySelectorAll(
                ".toolbarbutton-1, .bookmark-item"
              )
            )
          );
        }
```

fix some missing bookmarks